### DISCUSSION TITLE
Would the cell type proportion of the reference scRNA-seq dataset affect the performance or results of the deconvolution?

### DISCUSSION THREAD

**Discussion by ShaowenJ** [BayraktarLab/cell2location#243] [Q&A] [answered]
### Problem

So I am working on a slide-seq dataset on mouse lymph nodes tissue. And try to use cell2location to deconvolute the different cell types. I apply a similar strategy as in your paper of combining multiple scRNA-seq datasets to get a more comprehensive reference. I don't have or couldn't find a mouse LN scRNA-seq dataset, so I combine Spleen and Thymus as the reference instead. But as you probably know, Thymus has a large proportion of cells that shouldn't exist in the LN tissue, like Double-Negative, Double-Positive T cells. So do you think removing the unwanted cell types from the reference would be a good strategy or this would affect the results?

### Description of the data input and hyperparameters

Slide-seq V2 data

#### Single cell reference data: technology type (e.g. mix of 10X 3' and 5')

10X 

#### Spatial data: number of locations numbers, technology type (e.g. Visium, ISS, Nanostring WTA)

...

**Comment by vitkl** [comment, marked as answer]
Hi @Lesdormis 

Or reviewers asked a similar question - so posting it here - I think this outlines the main considerations related to reference data / spatial data mismatch

It is instructive to split this problem into distinct regimes: 1) low cell/nuclei numbers of certain cell types due to reduced isolation efficiency; 2) absence of specific cell types, that are captured in the spatial data, from the scRNA reference; 3) disassociation-related biases of gene expression signatures of cell types identified in the single cell RNA-seq. We discuss each of these settings in turn:
1. By design, cell abundance estimation by cell2location is not directly affected by the relative abundances of individual cell types in the scRNA data - only the average expression of each gene in each cell type is used for mapping. Thus, as long as a sufficient number of cells of a given type are captured to estimate the reference expression signature, the corresponding cell type can be mapped. For example, as few as 43 cells can be sufficient for mapping a cell type from a 10X 3’ v3 reference (e.g. Astro THAL hab subtype, Fig 3A, Fig 3E), while 4-15 cells profiled per cell type can be sufficient for a Smart Seq V4 reference (e.g. most of the Yao et al cell types, Fig 2K-L, Fig S11A).
2. The case of missing all major cell types that are very abundant in-situ, such as mapping exclusively immune cells to epithelial tissues. While such missing cell types cannot be mapped, cell2location is robust to this type of missing information and will still be able to map the remaining cell types. To demonstrate this, we considered the benchmark using simulated data, when removing reference expression signatures of increasing fractions of cell types. Notably, the accuracy of mapping of remaining cell types was retained as increasingly more neuron subtypes are removed from the reference (synthetic data, Fig S4D). Additionally, we note that in the lymph node data, the reference we constructed has a limited representation of stromal cells, however the mapping of B and T cells was still highly consistent with expected structure of the tissue (GC zones, T-cell zones, B-follicles, Fig 4C). 
3. Finally, tissue dissociation can introduce biases (https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1906-x) that can confound the estimation of reference expression signatures, in particular when transcriptional responses to dissociation are cell-type specific. The considered analysis using the lymph node dataset, which is combined from 3 studies, exemplifies that cell2location can cope with imperfect references. The cell2location model also has additional robustness to account for non-matching expression patterns (the technology difference scaling parameter m_g, and NB overdispersion). Nevertheless, there will be limitations and we cannot rule out that closely related cell types could be confounded by dissociation-related biases. Improvements to explicitly model such biases could be considered in the future.

**Reply by ShaowenJ** [reply]
Hi @vitkl , thanks for this detailed information! I would also discuss this with my colleagues who are all very interested in using cell2location for their analysis! And we probably would have some further questions to follow up.
Thanks again!
